.travis.yml

```diff
@@ -9,9 +9,9 @@ before_script:
 - npm prune
 script:
 - npm run build
-- if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then npm run test:ci; fi
+- npm run test
 after_success:
-- npm run semantic-release
```

Comment:
I'm guessing this is going to be added back in once the tests have been fixed and that you've removed this just so you can trigger builds/tests without performing a new release. Just wanted to flag that this is a required part of the build/release process.

Reply:
I will revert the `.travis.yml` before the proper commit, I wanted to make sure my builds on this PR were not releasing anything...